playbooks/roles/caasp4/tasks/setup_nodes.yml

```diff
@@ -56,6 +56,19 @@
     src: inventory-caasp4.yml.j2
     dest: "{{ socok8s_workspace }}/inventory/caasp4.yml"
 
+- name: Extend inventory to co-use the LB node as OSH deployer node
+  copy:
```

Comment:
also, this works in parallel in the CI with the real deployer, so there may be file conflicts writing to the soc-deployer.yml file

Reply:
Hm, good point.  I think I need to skip this when `deployment_mechanism==openstack`